app/services/remote_job_handler.rb

```diff
@@ -66,6 +69,43 @@ def set_submitted_to_if_necessary(job)
     end
   end
 
+  def execute_local_pre_process(job)
+    return unless job.executable.local_pre_process_script.present?
+    Dir.chdir( job.dir ) {
```

Comment:
This is an intensional behavior. All the output files of local preprocess are updated by the output files of the simulator.